connector/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -447,8 +449,9 @@ class KafkaTestUtils(
     sendMessages(msgs.toImmutableArraySeq)
   }
 
+  private val cleanupLogsPrivateMethod = PrivateMethod[LogManager](Symbol("cleanupLogs"))
   def cleanupLogs(): Unit = {
```

Comment:
In v3.8.0, the access modifier of method `cleanupLogs` has been changed to `private`

- v3.7.1
https://github.com/apache/kafka/blob/3.7.1/core/src/main/scala/kafka/log/LogManager.scala#L1390
<img width="594" alt="image" src="https://github.com/user-attachments/assets/43c1ae2b-8e1f-4205-99e9-585a5e71485d">

- v3.8.0
https://github.com/apache/kafka/blob/3.8.0/core/src/main/scala/kafka/log/LogManager.scala#L1395
<img width="596" alt="image" src="https://github.com/user-attachments/assets/5735612b-73db-4334-b7ed-6a50e1095896">